src/assets/scripts/client/aircraft/FlightManagementSystem/RouteModel.js

```diff
@@ -473,6 +484,32 @@ export default class RouteModel extends BaseModel {
         return routeString.replace(REGEX.DOUBLE_DOT, ' ').replace(REGEX.SINGLE_DOT, ' ');
     }
 
+    /**
+     * Returns exit waypoint for a departure aircraft, to be used in datablock
+     *
+     * When a SID procedure is defined, this will return the exit waypoint
+     * Example:
+     * - `KLAS07R.BOACH6.TNP` -> `TNP`
+     *
+     * When no SID procedure is defined, this will return the first fix in the route
+     * Example:
+     * - `OAL..MLF..PGS` -> `OAL`
+     *
+     * @for RouteModel
+     * @method getFlightPlanEntry
+     * @returns {string} First fix in flightPlan or exit fix of SID
+     */
+    getFlightPlanEntry() {
+        if (!this.hasSidLeg()) {
+            return this.getFullRouteString().split('..')[0];
+        }
+
+        const routeStringParts = this._divideRouteStringIntoSegments(this.getFullRouteString());
+        const temporaryLegModel = new LegModel(routeStringParts[0]);
+
+        return temporaryLegModel.getExitFixName();
```

Comment:
As mentioned by @felixscheffer , suggest:
```suggestion
        const sidLegIndex = this._findSidLegIndex();
        const sidLegModel = this._legCollection[sidLegIndex];
        
        return sidLegModel.getExitFixName();
```